src/views/MapView.vue

```diff
@@ -1,54 +1,53 @@
 <template>
   <div>
-    <div class="header" v-if="showHeader">
-      <span class="title">
-        <h1>{{ this.header.headerVal }}</h1>
-        <span class="caret-icon" @click="showHeader=false">
-          <h3 id="hide-title-tt">^</h3>
-        </span>
-        <b-tooltip target="hide-title-tt" triggers="hover" placement="bottom">
-          Hide Title
-        </b-tooltip>
-      </span>
-      <p class="sub-title">{{ this.header.subTitle }}</p>
-    </div>
+    <page-title :returnButton="true" v-if="showHeader" :subtitle="this.header.subTitle" title="">
+      <h3 id="hide-title-tt" class="caret-icon" @click="showHeader=false">Reserve New Block ^</h3>
+      <b-tooltip target="hide-title-tt" triggers="hover" placement="bottom">
+        Hide Title
+      </b-tooltip>
+    </page-title>
     <div v-else>
-      <p class="show-text" @click="showHeader=true">show</p>
+      <b-button class="show-text" @click="showHeader=true">show</b-button>
     </div>
-
-    <hr />
+    <b-tooltip target="hide-title-tt" triggers="hover" placement="bottom">
+      Hide Title
+    </b-tooltip>
     <div class="action-row">
       <SelectedStreets
-          class="streets-container"
           v-if="reservedFilter === 0"
           v-bind:onClick="reserveStreets"
           v-bind:streets="streetsToReserve"
           v-bind:setBlocks="setReserveStreets"
           v-bind:title="'Reserve'"/>
       <SelectedStreets
-          class="streets-container"
           v-if="(reservedFilter === 1) || isAdminMap"
           v-bind:onClick="unreserveStreets"
           v-bind:streets="streetsToUnreserve"
           v-bind:setBlocks="setUnreserveStreets"
           v-bind:title="'Unreserve'"/>
       <SelectedStreets
-          class="streets-container"
           v-if="(reservedFilter === 1) || isAdminMap"
           v-bind:onClick="completeStreets"
           v-bind:streets="streetsToComplete"
           v-bind:setBlocks="setCompleteStreets"
           v-bind:title="'Complete'"/>
+      <h3 v-if="reservedFilter === 1 && !!activeStreetId">Block {{this.activeStreetId}}</h3>
+      <b-button v-on:click="labelsVisible = !labelsVisible">
+        {{`${(this.labelsVisible ? "Hide" : "Show")} block labels`}}
+      </b-button>
     </div>
-
-      <Map
-        v-if="showHeader"
-        class="map-container-small"
-        v-bind:reservedFilter="this.reservedFilter"
-        v-bind:pushStreet="this.pushStreet"
-        v-bind:isAdminMap="this.isAdminMap"
-        v-bind:activeStreetId="this.activeStreetId"
-        ref="map"/>
+    <div class="header-bar">
+      <!-- <b-button v-if="reservedFilter === 0" disabled>Available blocks</b-button> -->
+      <!-- <b-button v-if="reservedFilter === 0" disabled>Blocks near me</b-button> -->
```

Comment:
Let's not commit comments into the mainline

Reply:
ok. removed this div and the buttons inside, they weren't being used.